### ISSUE TITLE
image-placeholder not found

### ISSUE THREAD

**Issue by siHopp-oowv** [quarto-dev/quarto-cli#10258]
### Bug description

Hi, 
I recently upgraded from Quarto 1.3.433 to Quarto 1.4.555. 
I have a larger Quarto project with a blog listing where I defined an image-placeholder. When rendering with the new quarto version, quarto says that it can't find the placeholder image even though I didn't move or rename it. I tried using different images in the same folder and some work and some produce the same error. I can't find anything in common between the ones that work and the ones that don't work. 

### Steps to reproduce

The header of the file with the listing:
```---
title: ""
sidebar: false
toc: false
back-to-top-navigation: false
format: 
  html:
    page-layout: full

listing:
    id: recent-blog-listing
    contents: "/blog/posts"
    sort: "date desc"
    categories: false
    type: grid
    max-items: 3
    image-placeholder: "_img/test_picture.jpg"
    #image-height: 250px
--- ```

![Uploading test_picture.jpg…]()

### Expected behavior

_No response_

### Actual behavior

GET: /
  /_img/test_picture.jpg (404: Not Found)

### Your environment

_No response_

### Quarto check output

```
Quarto 1.4.555
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.555
      Path: C:\Program Files\RStudio\resources\app\bin\quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: v2023.12
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\Users\z08975\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2023

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.12.0
      Path: C:/Python_3.12_x64/python.exe
      Jupyter: 5.7.2
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.4.1
      Path: C:/PROGRA~1/R/R-44~1.1
      LibPaths:
        - C:/Users/z08975/AppData/Local/R/win-library/4.4
        - C:/Program Files/R/R-4.4.1/library
      knitr: 1.48
      rmarkdown: 2.27

[>] Checking Knitr engine render......OK
```

**Comment by cscheid** [comment]
Sorry, but we need a minimal reproduction to be able to help.

**Comment by siHopp-oowv** [comment]
I can try to set up a minimal reproducible example tomorrow, but I figured out some more details: 

It seems to me that in the current quarto version, images only referred to in the YAML are not copied to the output directory. If I use images as placeholders that are embedded somewhere in the markdown code it works. If they are not used in the markdown it doesn't.

Is that already enough information for you to reproduce the problem?

**Comment by mcanouil** [comment]
We really need a fully reproducible example (as small as possible) for us to investigate.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by cscheid** [comment]
> Is that already enough information for you to reproduce the problem?

No, it's not. We really do need the _specific document that's failing for you_. I apologize, but we simply cannot justify setting aside an hour of our time to create an example that might or might not reproduce. It truly isn't personal: there's just too many of you and too few hours in our day.

**Comment by siHopp-oowv** [comment]
I tried to set up a reproducible example. 
I had not idea how to reduce the problem to a single file, because the bug only occurs in a listing. If you have an idea to keep the example simpler, I would be curious to learn how to. 
[repex_listing.zip](https://github.com/user-attachments/files/16162749/repex_listing.zip)

**Comment by mcanouil** [comment]
Indeed, if the resource is only set in `image-placeholder` Quarto does not copy it to the output location.

A workaround in theses cases is to explicitly tell to Quarto about those resources.

```yml
project:
  type: website
  resources:
    - _img
```
